### PR TITLE
Edit knowledge.md

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: knowledge.md [hunk 1]
```diff
@@ -20,14 +20,27 @@
 ### Web Application ###
 4. Describe what SQL Injection is and how you would test for it? *@jstnkndy*
 5. What about Blind SQL Injection and how is it different from other kinds? *@jstnkndy*
-7. Describe a webshell and how you would upload/use one.
+6. How can SQL Injection lead to remote code execution? *@morgoroth*
+7. How can you execute os command with mssql injection? *@enddo*
+7. Describe a webshell and how you would upload/use one. *@enddo*
+    1. How would you bypass uploader protections? *@enddo*
+8. Describe a Remote Command Execute(RCE). *@enddo*
```
Comment (WebBreacher) [inline knowledge.md]: This should probably be: Describe Remote Command Execution (RCE).

File: knowledge.md [hunk 2]
```diff
@@ -20,14 +20,27 @@
 ### Web Application ###
 4. Describe what SQL Injection is and how you would test for it? *@jstnkndy*
 5. What about Blind SQL Injection and how is it different from other kinds? *@jstnkndy*
-7. Describe a webshell and how you would upload/use one.
+6. How can SQL Injection lead to remote code execution? *@morgoroth*
+7. How can you execute os command with mssql injection? *@enddo*
+7. Describe a webshell and how you would upload/use one. *@enddo*
+    1. How would you bypass uploader protections? *@enddo*
+8. Describe a Remote Command Execute(RCE). *@enddo*
+    1. How would you perevent it in php? *@enddo*
```
Comment (WebBreacher) [inline knowledge.md]: Please capitalize PHP

File: knowledge.md [hunk 3]
```diff
@@ -20,14 +20,27 @@
 ### Web Application ###
 4. Describe what SQL Injection is and how you would test for it? *@jstnkndy*
 5. What about Blind SQL Injection and how is it different from other kinds? *@jstnkndy*
-7. Describe a webshell and how you would upload/use one.
+6. How can SQL Injection lead to remote code execution? *@morgoroth*
+7. How can you execute os command with mssql injection? *@enddo*
```
Comment (WebBreacher) [inline knowledge.md]: Please capitalize OS or spell out operating system

File: knowledge.md [hunk 4]
```diff
@@ -20,14 +20,27 @@
...
+    1. How would you exploit XSS?
 3. What is the purpose of the same origin policy with relation to the document object model? *@jstnkndy*
 4. Describe the basics of input and output of a block cipher. *@jstnkndy*
 5. How does the Heartbleed vulnerability work? *@webbreacher*
 6. How do you exploit the Shellshock vulnerability and what can an attacker do with it? *@webbreacher*
 
+### Exploit Development ###
+1. Describe what Buffer overflow is and how you would test for it? *@enddo*
+2. Describe what SEH is and how you exploit it? *@enddo*
+3. How debugger modules and plugins can speed up basic exploit development? *@enddo*
```
Comment (WebBreacher) [inline knowledge.md]: This needs something like "Describe how..." at the beginning

File: knowledge.md [hunk 5]
```diff
@@ -20,14 +20,27 @@
...
+4. How would you bypass DEP or ASLR in windows 7? *@enddo*
```
Comment (WebBreacher) [inline knowledge.md]: Please capitalize Windows

**Comment by ghost** [comment]
Thanks. I've corrected all the items that you mentioned.

**Pull request merged**